modules/account-lib/package.json

```diff
@@ -28,6 +28,7 @@
     "node": ">=14 <17"
   },
   "dependencies": {
+    "@bitgo/sdk-coin-ada": "^1.1.0",
```

Comment:
Only needed in `bitgo`. Can remove from `account-lib`.

Reply:
@ewangbitgo Can you confirm that we need sdk-coin-ada in account lib?